src/libs/actions/Policy/Policy.ts

```diff
@@ -2531,14 +2531,38 @@ function removePolicyReceiptPartnersConnection(policyID: string, partnerName: st
         },
     ];
 
+    const successData: OnyxUpdate[] = [
```

Comment:
Oh now that I realize this, we don't need the success since it is the same as the optimistic